### ISSUE TITLE
No need to softmax() for nn.CrossEntropyLoss()

### ISSUE THREAD

**Issue by whubaichuan** [GiorgiaD/PEPITA#1]
hi, in [Tutorial_PEPITA_FullyConnectedNets_CIFAR-10.ipynb](https://github.com/GiorgiaD/PEPITA/blob/main/Tutorial_PEPITA_FullyConnectedNets_CIFAR-10.ipynb), I notice that you do the softmax() for the network's output and then use this output to nn.CrossEntropyLoss(). However, [nn.CrossEntropyLoss()](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html) is equivalent to the combination of LogSoftmax and NLLLoss, which means nn.CrossEntropyLoss() will do the softmax() again.
